projects/cn0501/common/cn0501_bd.tcl

```diff
@@ -0,0 +1,71 @@
+
+# ad7768 interface
+
+create_bd_port -dir I adc_clk
+create_bd_port -dir I adc_valid
+create_bd_port -dir I adc_sync
+create_bd_port -dir I -from 31 -to 0 adc_data
+create_bd_port -dir I -from 31 -to 0 adc_gpio_0_i
+create_bd_port -dir O -from 31 -to 0 adc_gpio_0_o
+create_bd_port -dir O -from 31 -to 0 adc_gpio_0_t
+create_bd_port -dir I -from 31 -to 0 adc_gpio_1_i
+create_bd_port -dir O -from 31 -to 0 adc_gpio_1_o
+create_bd_port -dir O -from 31 -to 0 adc_gpio_1_t
+
+create_bd_intf_port -mode Master -vlnv xilinx.com:interface:iic_rtl:1.0 iic_0_io
+
+# instances
+
+ad_ip_instance axi_dmac ad7768_dma
+ad_ip_parameter ad7768_dma CONFIG.DMA_TYPE_SRC 2
+ad_ip_parameter ad7768_dma CONFIG.DMA_TYPE_DEST 0
+ad_ip_parameter ad7768_dma CONFIG.CYCLIC 0
+ad_ip_parameter ad7768_dma CONFIG.SYNC_TRANSFER_START 1
+ad_ip_parameter ad7768_dma CONFIG.AXI_SLICE_SRC 0
+ad_ip_parameter ad7768_dma CONFIG.AXI_SLICE_DEST 0
+ad_ip_parameter ad7768_dma CONFIG.DMA_2D_TRANSFER 0
+ad_ip_parameter ad7768_dma CONFIG.DMA_DATA_WIDTH_SRC 32
+
+# ps7-hp1
+
+#ad_ip_parameter sys_ps7 CONFIG.PCW_USE_S_AXI_HP1 1
```

Comment:
Please delete commented out code.

Reply:
cleaned comment; I could use the pack for the cora but I was not sure how that works for de10nano so I used the module for both projects